### PR TITLE
AWS Health API high availability endpoint demo code

### PR DESCRIPTION
**Pull request by robertmaldon**
*Description of changes:*

Demo code for using the AWS Health API high availability endpoint, so that customers can determine and use the current active endpoint, and therefore always get the latest health events for their accounts.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: high-availability-endpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+high-availability-endpoint
+--------------------------
+
+This repository contains sample code for using the AWS Health API's high availability endpoint to determine which region to connect to in order to get the latest Health information.
+
+## Background
+
+AWS Health is a RESTful web service that uses HTTPS as a transport and JSON as a message serialization format. Your application code can make requests directly to the AWS Health API. When using the REST API directly, you must write the necessary code to sign and authenticate your requests. For more information, see the [AWS Health API Reference](https://alpha-docs-aws.amazon.com/health/latest/APIReference/).
```
Comment (tipuq) [inline high-availability-endpoint/README.md:8]: please replace with a GA doc link (not alpha)

Comment (tipuq) [inline high-availability-endpoint/README.md:8]: replace alpha docs link with GA doc link.

**Review by tipuq — CHANGES_REQUESTED** [review]
just need to update docs link

File: high-availability-endpoint/README.md (hunk 1 above)
Comment (robertmaldon) [reply high-availability-endpoint/README.md]: hi, 

Good catch!

I've changed the link, squashed the commit, and pushed the new commit as part of this PR.

**Pull request merged**